lib/plugins/bidirectionalData.js

```diff
@@ -114,9 +117,28 @@ function sendSubscriptions(device, attributeList, callback) {
         });
     }
 
+    function sendSingleSubscriptionNgsi2(item, innerCb) {
+        var variables = extractVariables(item);
+
+        subscriptions.subscribe(device, [item.name], variables, function handleSubscription(error, subId) {
```

Comment:
Maybe it would be better to call directly to subscribeNgsi2(). In the same fashinon, sendSingleSubscriptionNgsi1() should call directly to subscribeNgsi1()

As consecuence of both above, subscribe() splitter in subscriptionService.js would be unnecessary.

Reply:
It would a valid option but in that case, we should modify the functions that are exported by https://github.com/telefonicaid/iotagent-node-lib/blob/master/lib/services/ngsi/subscriptionService.js#L207.

I have tried to follow just the style that you proposed for instance in https://github.com/dcalvoalonso/iotagent-node-lib/blob/task/activeAttsNgsi2/lib/services/ngsi/ngsiService.js#L467

Reply:
Ok, no problem. The NGSIv1/NGSIv2 split code should be transient (although maybe a long transient ;) so it is not so important. NTC.